eng/pipelines/templates/jobs/sign-and-pack.yml

```diff
@@ -117,4 +125,19 @@ jobs:
 - template: /eng/pipelines/templates/jobs/vsix/pack-and-sign-vsix.yml
   parameters:
     DependsOn: Sign
-    ServerName: ${{ parameters.ServerName }}
+
+- template: /eng/pipelines/templates/jobs/docker/stage-docker.yml
+  parameters:
+    DependsOn: Sign
+
+- ${{ each os in split('linux|windows|macOS', '|') }}:
```

Comment:
Putting a pin in this. I believe that Dozie is handling the different name of the OS in the build matrix _in_ `verify-packages.yml`.

Will verify and come back to this.

Reply:
He's expecting a matrix file and the generate-matrix job.  We're creating all of the matrices at the start of pipeline instead of using a static matrix file now.